src/management/projects.ts

```diff
@@ -411,6 +417,19 @@ export async function getFirebaseProject(projectId: string): Promise<FirebasePro
     });
     return res.body;
   } catch (err: any) {
+    if (getErrStatus(err) === 404) {
+      try {
+        const info = await getProject(projectId);
+        // TODO: Update copy based on Rachel/Yvonne's feedback.
```

Comment:
This messaging is coming later as part of supporting projects that don't have firebase enabled.

Reply:
Consider putting a debug log in here, just in case someone is having _some_ issue, you'll know this path is being followed (and they're not seeing messaging they should be)

Reply:
Good call, done